docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,26 @@
+# Tizen IoT with Nubison Cloud
+
+The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an instrinsic capability of seamless connectivity.
+The Nubison cloud empowers the users to send or receive requests for Tizen service applications in real-time with ease.
```

Comment:
Just a question. Is "send or receive requests" correct? The original sentence is "send data or receive requests."
Regarding the below image, it looks like from IoT Device user send "Data" and get "Controlling" from Nubison Cloud, so I'm not sure with "send or receive requests".

Reply:
@mijin85cho  thanks for your question. The original sentence is "send data or receive requests."
Right
Generally in IoT paradigm, data is always sent as a part of request so I changed it to send or receive requests.

Reply:
Our team had a discussion on this and came up with the following sentence:

Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.

Reply:
@bson1012 , the sentence seems fine to me , with more detail. slight change in sentence:
Kindly update it with: 

Real-time data monitoring and controlling between Tizen service app and Nubison cloud is commissioned using Nubison plugin.